src/Service/OrbitService.cpp

```diff
@@ -149,6 +159,8 @@ void OrbitService::Run(std::atomic<bool>* exit_requested) {
   LOG("**********************************");
 #endif
 
+  PrintClockResolution();
```

Comment:
I would recommend checking that the resolution is some minimum sane resolution, and if it is not, at least warn the user. For example, if the resolution is as low as 1ms then you maybe don't want to even attempt profiling. 
(As it seems to has happened with introspection on Windows, which was the motivation for this change, if I understood correctly?)

Reply:
Good suggestion, and I think we should do that. We currently don't have a direct way of sending warnings/errors from the service to the UI to be displayed, so this needs to be discussed first how we  want to do that. As for now, I'll not address this in the current PR.